### PR TITLE
Kraken: fetch prices from public API

### PR DESCRIPTION
**Pull request by hsanjuan**
Also small refactor to handle request code all in the same place.

### CODE REVIEW

**Comment by frrist** [comment]
@hsanjuan did you mean to assign or are you requesting review?

**Comment by hsanjuan** [comment]
Sorry @frrist , it was late. I was requesting review :)

**Review by iand — APPROVED** [review]

File: quotetracker/coinmarketcap.go [hunk 1]
```diff
@@ -2,35 +2,60 @@ package quotetracker
...
-type curData struct {
-	Quote map[string]*quoteData `json:"quote"`
+type coinMarketCapData struct {
+	Quote map[string]*coinMarketCapQuote `json:"quote"`
 }
 
-type quoteData struct {
+type coinMarketCapQuote struct {
 	Price     float64 `json:price`
 	Volume24h float64 `json:volume_24h`
```
Comment (frrist) [inline quotetracker/coinmarketcap.go:31]: should these be quoted?
```suggestion
type coinMarketCapQuote struct {
	Price     float64 `json:"price"`
	Volume24h float64 `json:"volume_24h"`
}
```

Comment (hsanjuan) [reply quotetracker/coinmarketcap.go:31]: oops. yes

Comment (hsanjuan) [reply quotetracker/coinmarketcap.go:31]: 37dc827

**Pull request merged**